app/core/models.py

```diff
@@ -0,0 +1,24 @@
+from django.db import models
+
+
+class Recipe(models.Model):
+    """Recipe object"""
+    name = models.TextField(blank=False)
+    description = models.TextField()
+
+    def __str__(self):
+        return self.name
+
+
+class Ingredient(models.Model):
+    """Ingredient Object"""
+    name = models.TextField()
+    recipe = models.ForeignKey(
+        Recipe,
+        on_delete=models.CASCADE,
+        related_name='ingredients',
+        null=True,
```

Comment:
An ingredient should always belong to a recipe, so you shouldn't allow `null=True`.